README.md

```diff
@@ -151,6 +178,41 @@ render(
 )
 ```
 
+#### Configuring vite-plugin with SolidJS
+
+Remember to check the `dirs` is set to the correct routes directory in `vite.config.ts` or `app.config.ts`:
+
+*vite.config.ts* (without SolidStart)
+
+```ts
+import { defineConfig } from 'vite';
+import solidPlugin from 'vite-plugin-solid';
+import Pages from 'vite-plugin-pages';
+
+
+export default defineConfig({
+  plugins: [Pages({
+    dirs: ['src/pages'],
+  }), solidPlugin()],
+});
+```
+
+*app.config.ts* (with SolidStart)
```

Comment:
Why would people using SolidStart with this plugin? Is there any use case for this one?

Reply:
That's a very fair point. To be honest, there's a process ongoing right now to replace vinxi entirely (because it's a bit obsoleted by vite v6 + nitro v3), and that might even result in a sunset of the app.config.ts in favor of just a vite.config.ts, so I've removed the app.config.ts here for now.